cassandra-mesos-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/scheduler/PersistedCassandraFrameworkConfiguration.java

```diff
@@ -43,7 +43,9 @@ public PersistedCassandraFrameworkConfiguration(
         @NotNull final String mesosRole,
         @NotNull final String dataDirectory,
         final boolean jmxLocal,
-        final boolean jmxNoAuthentication
+        final boolean jmxNoAuthentication,
+        final String defaultRack,
+        final String defaultDc
```

Comment:
Please add `@NotNull` to these fields since there is a defined default, and it will be necessary for the scheduler to ensure these values are always provided.